apps/condo/domains/organization/integrations/sbbol/sync/index.js

```diff
@@ -101,8 +105,15 @@ const sync = async ({ keystone, userInfo, tokenSet, reqId }) => {
     await sbbolSecretStorage.setOrganization(organization.id, user.id)
     await syncTokens(tokenSet, user.id)
     await syncSubscriptions()
+    const bankIntegrationContextId = get((await checkSbbolBankIntegrationContext(context.context, organization.id))[0], 'id')
```

Comment:
`context.context` looks awful ;)
Please, get rid of `context.context`, try to refactor its usage in another functions.